src/Nethermind/Nethermind.Runner/configs/gnosis.cfg

```diff
@@ -23,6 +23,24 @@
     "SecondsPerSlot": 5,
     "TargetBlockGasLimit": 17000000
   },
+  "Shutter": {
+    "Validator": "true",
+    "ValidatorRegistryContractAddress": "0xefCC23E71f6bA9B22C4D28F7588141d44496A6D6",
+    "SequencerContractAddress": "0xc5C4b277277A1A8401E0F039dfC49151bA64DC2E",
+    "KeyBroadcastContractAddress": "0x626dB87f9a9aC47070016A50e802dd5974341301",
+    "KeyperSetManagerContractAddress": "0x7C2337f9bFce19d8970661DA50dE8DD7d3D34abb",
+    "KeyperP2PAddresses": [
+      "/ip4/139.59.130.109/tcp/23003/p2p/12D3KooWRZoofMsnpsjkgvfPQUyGXZQnn7EVnb4tw4ghNfwMnnsj",
+      "/ip4/167.71.169.248/tcp/23003/p2p/12D3KooWGH3VxoSQXZ6wUuCmsv5caGQnhwfGejbkXH6uS2r7sehA",
+      "/ip4/139.59.130.109/tcp/23003/p2p/12D3KooWNxTiw7CvD1fuyye5P8qPhKTTrRBW6wwZwMdqdTxjYF2H",
+      "/ip4/178.128.192.239/tcp/23003/p2p/12D3KooWCdpkipTiuzVMfkV7yLLgqbFeAL8WmEP78hCoBGBYLugN",
+      "/ip4/45.55.192.248/tcp/23003/p2p/12D3KooWMPuubKqksfMxvLwEBDScaopTdvPLr5J5SMmBEo2zkcMz",
+      "/ip4/178.128.126.237/tcp/23003/p2p/12D3KooWAg1pGUDAfFWSZftpN3JjBfLUCGLQcZApJHv2VntdMS9U"
+    ],
+    "P2PPort": "23102",
```

Comment:
P2PPort and EncryptedGasLimit should probably be this by default? Remove all defaults from cfg files.